src/fmeval/transforms/common.py

```diff
@@ -191,3 +195,76 @@ def __call__(self, record: Dict[str, Any]) -> Dict[str, Any]:
         avg = np.mean([record[input_key] for input_key in self.input_keys])
         record[self.output_key] = avg
         return record
+
+
+class BertScoreMax(Transform):
+    """This Transform augments its input record with the maximum BERTScore metric computed over
+    possible targets separated by a target_output_delimiter.
+    """
+
+    def __init__(
+        self,
+        target_output_keys: List[str],
+        model_output_keys: List[str],
+        output_keys: List[str],
+        allow_duplicate_input_keys: bool,
+        bertscore_model: Union[BertscoreHelperModel, ActorHandle],
```

Comment:
I originally made a `split_function` parameter instead of the `target_output_delimiter` in an attempt to make the component have a more generic use case, but I ran into issues in the `test_qa_accuracy_evaluate` function with ray. I will start debugging and see if I can make this work, but I reverted back to the `target_output_delimiter` for now. 
<img width="1512" alt="Screenshot 2024-07-25 at 3 25 18 PM" src="https://github.com/user-attachments/assets/bb8227eb-055c-43fe-bb9a-6d8d5600017a">